packages/phrases-ui/src/locales/zh-tw/description.ts

```diff
@@ -51,18 +51,12 @@ const description = {
   no_region_code_found: '沒有找到區域碼',
   verify_email: '驗證你的郵箱',
   verify_phone: '驗證你的手機號碼',
-  /** UNTRANSLATED */
-  password_requirements: 'Password {{items, list}}.',
-  /** UNTRANSLATED */
-  'password_requirement.length_one': 'requires a minimum of {{count}} character',
-  /** UNTRANSLATED */
-  'password_requirement.length_other': 'requires a minimum of {{count}} characters',
-  /** UNTRANSLATED */
-  'password_requirement.character_types_one':
-    'should contain at least {{count}} type of uppercase letters, lowercase letters, digits, and symbols',
-  /** UNTRANSLATED */
+  password_requirements: '密碼{{items, list}}。',
+  'password_requirement.length_one': '長度至少為{{count}}個字符',
+  'password_requirement.length_other': '長度至少為{{count}}個字符',
+  'password_requirement.character_types_one': '需包含{{count}}類型的大寫字母、小寫字母、數字和符號',
   'password_requirement.character_types_other':
-    'should contain at least {{count}} types of uppercase letters, lowercase letters, digits, and symbols',
+    '需包含{{count}}類型的大寫字母、小寫字母、數字和符號',
```

Comment:
sometimes the spacing (e.g. 需包含 2 類) works but sometimes it doesn't :-( nothing to do here i just wanted to complain ChatGPT

Reply:
Yeah... I also clearly stated in the prompt that we need to make sure there is a space between CJK characters and non-CJK ones. But it just won't listen... And it feels like the more I tell it to do, the less it will listen.

Reply:
Ah, but this case is zh-TW, and it makes sense to not including the spaces around the curly brackets, because CJK by default do not have spaces in a sentence.

Reply:
> CJK by default do not have spaces in a sentence

[not sure](https://github.com/sparanoid/chinese-copywriting-guidelines#%E4%B8%AD%E6%96%87%E8%88%87%E6%95%B8%E5%AD%97%E4%B9%8B%E9%96%93%E9%9C%80%E8%A6%81%E5%A2%9E%E5%8A%A0%E7%A9%BA%E6%A0%BC) if this is true - anyways. we can proceed

Reply:
I mean, this is true if the dynamic content is latin character or digital number, but false if the content is CJK characters. ChatGPT is not small enough to know the interpolation result based on the given context. So like I said we can't blame it on this.
Still need to manually improve some of these AI translated results if we want it to be perfect fit.